core/src/main/scala/com/evolutiongaming/catshelper/FromFuture.scala

```diff
@@ -14,12 +14,25 @@ trait FromFuture[F[_]] {
 
 object FromFuture {
 
+  /** Convinient summon-and-run function
+   *
+   * @example {{{
+   *          class Foo[F[_]: FromFuture: Monad](db: FutureDB){
+   *            def query(lines: Int): F[Unit] = FromFuture.defer[F](db.query(lines))
+   *          }
+   * }}} */
+  def defer[F[_]]: FromFutureDeferPA[F] = new FromFutureDeferPA[F]()
+
+  class FromFutureDeferPA[F[_]](val __ : Boolean = true) extends AnyVal {
+    def apply[A](future: => Future[A])(implicit instance: FromFuture[F]): F[A] = instance(future)
```

Comment:
usually we call this `F`, just an advice to rename `instance` to `F`, not a review blocker though